urnai/base/savable.py

```diff
@@ -0,0 +1,225 @@
+import os
+import pickle
+import tempfile
+from multiprocessing import Process
+
+#import time
+#from urnai.utils.reporter import Reporter as rp
+
+
+class SavableAttr:
+    def __init__(self, value):
+        self.value = value
```

Comment:
I think we can remove this and use a `Dict` instead.

Reply:
I noticed that this class wasn't being used for anything, so i just removed it, ok?

Reply:
Yes.